features/admin/currency/managing_currencies/browsing_currencies.feature

```diff
@@ -8,8 +8,8 @@ Feature: Browsing currencies
         Given the store has currency "Euro", "British Pound"
         And I am logged in as an administrator
 
-    @todo @ui @api
+    @api @ui
     Scenario: Browsing currencies in store
         When I want to browse currencies of the store
-        Then I should see 2 currencies in the list
-        And I should see the currency "British Pound" in the list
+        Then I should see 2 currencies on the list
+        And I should see the currency "British Pound" on the list
```

Comment:
I assume that
```suggestion
        Then I should see 2 currencies in the list
        And I should see the currency "British Pound" on the list
```
is the most correct version of these steps. But `on the list` is actually the more common form, so it could stay that way 🤔 

Ref.: https://www.britannica.com/dictionary/eb/qa/is-it-correct-to-say-on-the-list-or-in-the-list